RLTest/env.py

```diff
@@ -157,7 +158,7 @@ def compareEnvs(self, env):
         return True
 
     def __init__(self, testName=None, testDescription=None, module=None,
-                 moduleArgs=None, env=None, useSlaves=None, shardsCount=None,
+                 moduleArgs=None, env=None, useSlaves=None, shardsCount=None, decodeResponses=True,
```

Comment:
So basically this can be set to None right? So if not set it gets the default?

Reply:
Yes, we can make that change! Right now we redundantly check the defaults to assign this to `True` later.